src/shims/fs.rs

```diff
@@ -139,6 +139,27 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             } else {
                 this.handle_not_found()
             }
+        } else if cmd == this.eval_libc_i32("F_DUPFD")? || cmd == this.eval_libc_i32("F_DUPFD_CLOEXEC")? {
+            // Note that we always assume the FD_CLOEXEC flag is set for every open file, in part
+            // because exec() isn't supported. The F_DUPFD and F_DUPFD_CLOEXEC commands only
+            // differ in whether the FD_CLOEXEC flag is pre-set on the duplicated file descriptor,
```

Comment:
```suggestion
            // differ in whether the FD_CLOEXEC flag is pre-set on the new file descriptor,
```
I was unsure for a second here if this referred to the input FD (that gets duplicated) or the output FD (that got duplicated).